### ISSUE TITLE
[MASTER] SQL fixes Human & Dwarven zone

### ISSUE THREAD

**Issue by Eldwarvo** [TrinityCore/TrinityCore#20177]
**Description:**
Fixed 3 quests in the Dwarven starting area, now you can pick up the items required for  
First Things First: We're Gonna Need Some Beer
A refugee's Quandary 
Pack your bags

2nd part, made human starting zone quests obtainable by otherAlliance races not native to Northshire

**Current behaviour:**
Can't pick up all items required for the quests
Can't get the quests in Northshire when you're a non human

**Expected behaviour:**
Being able to pickup the required items
Being able to pickup the quests in Northshire when you're a non-human

**Steps to reproduce the problem:**
Try to complete said quests
Try to create a nonhuman and pickup the human startingquests

**Branch(es):**

CHANGEME Master

**TC rev. hash/commit:** 
 1de03622c

**TDB version:**  TDB 720.00

 ```sql
-- Fix human startquests, make them obtainable for all alliance Races:
-- AllowableRaces = 1101;
update quest_template set AllowableRaces = 1101 where logtitle='Beating them back!';
update quest_template set AllowableRaces = 1101 where logtitle='Lions for Lambs';
update quest_template set AllowableRaces = 1101 where logtitle='Join the battle!';
update quest_template set AllowableRaces = 1101 where logtitle='They sent assassins';
update quest_template set AllowableRaces = 1101 where logtitle='The rear is clear';

**Comment by malcrom** [comment]
I see hacks

**Comment by Eldwarvo** [comment]
Care to explain?
Thanks

**Comment by Killyana** [comment]
Data fields in gameobject_template are WDB data and should not be changed.

**Comment by Eldwarvo** [comment]
Ok. Then why on the pickup quests only one object can be picked up on the original db, and after these changes (and checking on the description of the fields) it works?  For the objects that couldnt be picked up data30 was filled with the value that should have been in data1, and data1 was originally 0. Or maybe I did something  wrong because on none of my characters I was able to pick up more then one keg or Felixstowe thingies.
